rest_framework_api_key/models.py

```diff
@@ -21,7 +21,7 @@ def create_key(self, **kwargs) -> Tuple["APIKey", str]:
 
     def is_valid(self, key: str) -> bool:
         try:
-            prefix, _ = key.split(".")
+            prefix, _, _ = key.partition(".")
         except ValueError:
             return False
```

Comment:
We can remove the `try/except` block since  `str.partition()` never raises a `ValueError` (see https://docs.python.org/3/library/stdtypes.html#str.partition).

Reply:
2051a29